.gitignore

```diff
@@ -114,4 +114,5 @@ env/
 
 ### Configuration files ###
 .env
+.env.test
```

Comment:
If this is required for tests to run, it should not be gitignored! There shouldn't be any secrets in there.